autopush_rs/src/server/mod.rs

```diff
@@ -463,6 +463,20 @@ impl Server {
         Err("User not connected".into())
     }
 
+    /// A check for notification command has come for the uaid
+    pub fn check_client_storage(&self, uaid: Uuid) -> Result<()> {
+        let mut uaids = self.uaids.borrow_mut();
+        if let Some(client) = uaids.get_mut(&uaid) {
+            client
+                .tx
+                .unbounded_send(ServerNotification::CheckStorage)
```

Comment:
(in my quest to kill all our unwraps) I think .unbounded_send()? should work here instead of unwrap

Reply:
Ah, was just copy/pasting from the existing notify_client for the most part, which did the borrow_mut and such as well.

Reply:
Using ? here doesn't work as there's no From trait that satisfies the Result that unbounded_send returns, but using expect() does work fine.

Reply:
Course, expect panics.... so I'm just going to test the result and handle it properly. :)

Reply:
> Using ? here doesn't work as there's no From trait that satisfies the Result that unbounded_send returns, but using expect() does work fine.

ah yes. we could alternatively add futures::sync::mpsc::SendError<ServerNotification> to error_chain! to get the From trait for this to work